src/@ionic-native/plugins/phonegap-plugin-local-notification/index.ts

```diff
@@ -0,0 +1,106 @@
+import { Injectable } from '@angular/core';
+import { Cordova, CordovaInstance, Plugin, IonicNativePlugin, checkAvailability } from '@ionic-native/core';
+
+declare var Notification: any;
+// can use a shorter name here, like PLNObject ?
+export class PLNObject {
+
+  private _objectInstance: any;
+
+  constructor(title: string, options: any) {
+    if (checkAvailability('Notification') === true) {
+      this._objectInstance = new Notification(title, options);
+    }
+  }
+
+  @CordovaInstance({ sync: true })
+  close(): void { }
+
+}
+
+export interface LocalNotificationOptions {
+
+  /**
+   * Sets the direction of the notification. One of "auto", "ltr" or "rtl"
+   */
+  dir?: string;
+
+  /**
+   *  Sets the language of the notification
+   */
+  lang?: string;
+
+  /**
+   * Sets the body of the notification
+   */
+  body?: string;
+
+  /**
+   * Sets the identifying tag of the notification
+   */
+  tag?: string;
+
+  /**
+   * Sets the icon of the notification
+   */
+  icon?: string;
+
+}
+
+/**
+ * @name phonegap-local-notifications
```

Comment:
Name needs to be `Phonegap Local Notification` ... also directory name should be `phonegap-local-notification`